server/src/main/java/com/defold/extender/services/cocoapods/PodSpecParser.java

```diff
@@ -0,0 +1,754 @@
+                spec.osxModuleMap = createOsxModuleMap(spec, args.workingDir);
+            }
+        }
+        if (spec.osxModuleMap != null) {
+            spec.flags.osx.objc.add("-fmodule-map-file=" + spec.osxModuleMap);
+            spec.flags.osx.objcpp.add("-fmodule-map-file=" + spec.osxModuleMap);
+        }
+
+        return spec;
+    }
+
+    static JSONObject parseJson(String json) throws ExtenderException {
+        try {
+            JSONParser parser = new JSONParser();
+            return (JSONObject)parser.parse(json);
+        }
+        catch (ParseException e) {
+            e.printStackTrace();
+            throw new ExtenderException(e, "Failed to parse json. " + e);
+        }
+    }
+
+    // https://github.com/CocoaPods/Core/blob/master/lib/cocoapods-core/specification.rb#L187
+    static String getModuleName(JSONObject specJson, PodSpec parent) {
+        if (specJson.containsKey("module_name")) {
+            return (String)specJson.get("module_name");
+        }
+
+        if (specJson.containsKey("header_dir")) {
+            return toC99extIdentifier((String)specJson.get("header_dir"));
+        }
+
+        String name = (String)specJson.get("name");
+        String fixedName = toC99extIdentifier(name);
+        if (parent == null) {
+            return fixedName;
+        }
+
+        return parent.name + (NAME_EXCEPTIONS.contains(parent.name) ? "" : "_") + fixedName;
+    }
+    
+    static String toC99extIdentifier(String s) {
+        return s.replaceAll("^([0-9])", "_$1")  // 123FooBar -> _123FooBar
+            .replaceAll("[\\+].*", "")          // NSData+zlib -> NSData
+            .replaceAll("[^a-zA-Z0-9_]", "_")   // Foo-Bar -> Foo_Bar
+            .replaceAll("_+", "_");             // Foo__Bar -> Foo_Bar
+    }
+
+    static void parseMultiPlatformConfig(PodSpec spec, JSONObject config) {
+        if (config != null) {
+            parseConfig(config, spec.flags.ios, spec.linkflags.ios, spec.defines.ios);
+            parseConfig(config, spec.flags.osx, spec.linkflags.osx, spec.defines.osx);
+            JSONObject iosConfig = (JSONObject)config.get("ios");
+            JSONObject osxConfig = (JSONObject)config.get("ios");
+            if (iosConfig != null) parseConfig(iosConfig, spec.flags.ios, spec.linkflags.ios, spec.defines.ios);
+            if (osxConfig != null) parseConfig(osxConfig, spec.flags.osx, spec.linkflags.osx, spec.defines.osx);
+        }
+    }
+
+
+    static void parseConfig(JSONObject config, LanguageSet flags, Set<String> linkflags, Set<String> defines) {
+        // https://pewpewthespells.com/blog/buildsettings.html
+        // defines
+        List<String> defs = getStringListValues(config, "GCC_PREPROCESSOR_DEFINITIONS");
+        if (defs != null) {
+            defines.addAll(unescapeStrings(defs));
+        }
+        // linker flags
+        // https://xcodebuildsettings.com/#other_ldflags
+        List<String> ldFlags = getStringListValues(config, "OTHER_LDFLAGS");
```

Comment:
Main fix. Now value can be string with ' ' as delimiter or array of strings.
Also removed `$(inherited)` value which not a valid value for linker (it's valid for xcode config files).